gradle/verification-metadata.xml

```diff
@@ -160,6 +160,11 @@
             <sha256 value="06eba41a81aaccb011c3f75afa019e509cda7f1eb7a4e057bb860c60845f915e" origin="Verified"/>
          </artifact>
       </component>
+      <component group="com.github.spotbugs" name="spotbugs-annotations" version="4.8.6">
```

Comment:
All these new versions have been brought in once I have updated the `sonar-plugin-api`. The update was required since the version used did not have the `addTelemetryProperty` method in `SensorContext`.